<?xml version="1.0"?>

<launch>
  
  <!-- SINGABOAT-VRX Station-Keeping -->
  <node pkg="singaboat_vrx" type="singaboat_station_keeping.py" name="singaboat_station_keeping" output="screen"/>

  <!-- SINGABOAT-VRX Inverse Kinematics -->
  <node pkg="singaboat_vrx" type="singaboat_inverse_kinematics.py" name="singaboat_inverse_kinematics" output="screen"/>

</launch>
